src/federation-info-plugin/views/SchemaView/SchemaView.jsx

```diff
@@ -89,38 +61,61 @@ const SchemaView = ({ schemaViewData, rootTypes }) => {
       } else if (type.name === rootTypes.mutations) {
         mutations = type.fields
       } else if (type.name === rootTypes.subscriptions) {
-        subscriptions = type.subscriptions
+        subscriptions = type.fields
       } else {
         types.push(type)
       }
     }
+
     return { queries, mutations, subscriptions, types }
   }, [schemaViewData, rootTypes])
 
+  const sortQueries = (property, order) =>
```

Comment:
I have mixed feelings about this way of handling the sorting, it feels a bit convoluted, and it also has a very bad performance impact (although at this level it shouldn't be noticable yet)

Could we maybe make a util function that we use directly inside `SchemaFieldsTable` and possibly memoize it there?

So instead of passing `onSortChange={sortQueries}` we use util function `SchemaFieldsTable`.

Reply:
Commit pushed based on the solution we discussed on the call.